private/buf/bufworkspace/workspace_targeting.go

```diff
@@ -576,6 +599,20 @@ func getMappedModuleBucketAndModuleTargeting(
 	return mappedModuleBucket, moduleTargeting, nil
 }
 
+func isBucketEmpty(ctx context.Context, bucket storage.ReadBucket) (bool, error) {
```

Comment:
This can be replaced with [`storage.IsEmpty`](https://github.com/bufbuild/buf/blob/777f4fa5f17f1eb2343fbb85021384f87daaffec/private/pkg/storage/util.go#L161)